### ISSUE TITLE
There is no way to use the GPU.

### ISSUE THREAD

**Issue by y2695771481** [snehilchatterjee/respiratory_disease_classifier#15]
I was able to run your project successfully, but I noticed that the GPU utilization was 0. However, my CUDA is available and I tried to resolve the issue but was unsuccessful.
Can you help me clear up my confusion? Thank you very much!

**Comment by snehilchatterjee** [comment]
what does this part of the notebook print in your code (1st cell):
![image](https://github.com/user-attachments/assets/d1e2d4e2-e405-405a-8d94-3dfe8e1ecb2e)

**Comment by y2695771481** [comment]
![aa2c6e6777a542aa5ac2d972e4fe3c9](https://github.com/user-attachments/assets/bb566d16-1874-4ab2-b6ce-65647be300fc)
Please ask how long it takes you to complete a training (epochs=100), I successfully displayed Using cuda, but the GPU usage is still 0, the training time is very long。

**Comment by snehilchatterjee** [comment]
I don't remember exactly but I am pretty sure that I let it run over night. Moreover the reason your GPU utilization is low might have to do with the fact that models are very lightweight. During the training the CUDA was indeed working for me.

**Comment by snehilchatterjee** [comment]
Also you can use the weights I provided. You don't have to train it yourself.

**Comment by snehilchatterjee** [comment]
Hi if its working can I close this issue or is further help needed ?

**Comment by y2695771481** [comment]
May I ask if there is a corresponding paper for this project? If so, could you please provide information or links to relevant papers?

**Comment by snehilchatterjee** [comment]
No, I made this project just for learning how to work with audio data.

**Comment by snehilchatterjee** [comment]
Are you working on a research paper ?

**Comment by y2695771481** [comment]
我目前没有在写论文。但是我正在做这个领域的学习和研究。另外，非常感谢您耐心的回复。

**Comment by y2695771481** [comment]
I am not writing a paper at present. But I am doing learning and research in this field. In addition, I am really grateful for your patient reply.

**Comment by snehilchatterjee** [comment]
Oh yeah no worries brother. When I was working in the audio domain I referred to this youtube channel: https://youtube.com/@valeriovelardothesoundofai?feature=shared. You can check it once. It might be helpful.
